app/themes/dashboard/ui/app/navigation/navigation.js

```diff
@@ -69,15 +72,24 @@
       return service;
     }
 
-    TopNavCtrl.$injector = ['$scope', 'userService'];
-    function TopNavCtrl($scope, userService) {
-      var ctrl = this;
+  TopNavCtrl.$injector = ['$scope', 'userService', '$document'];
 
-      ctrl.currentUser = userService.getUser();
+  function TopNavCtrl($scope, userService, $document) {
+    var ctrl = this;
 
-      $scope.$watch(userService.currentUser, function(newValue) {
-        ctrl.currentUser = newValue;
-      });
-    }
+    ctrl.currentUser = userService.getUser();
+
+    $scope.$watch(userService.currentUser, function(newValue) {
+      ctrl.currentUser = newValue;
+    });
+
+    ctrl.toggleSidebar = function() {
+      var body = angular.element($document[0].body);
+      var states = { collapsed: 'sidebar-collapse', open: 'sidebar-open' };
+      var collapsed = body.hasClass(states.collapsed);
+      body.removeClass(collapsed ? states.collapsed : states.open);
+      body.addClass(collapsed ? states.open : states.collapsed);
+    };
+  }
```

Comment:
Yes, that is probably how it is used, but that runs on jquery directly. I'd prefer using angular, or at least hiding away jquery more..

A quick google turned up this, maybe worth a look? http://devartisans.com/articles/sidebar-navigation-using-angular

Check it out first, and see if you can find an angular way to make the sidebar work. I think it will look better. We can discuss offline before pushing too if you like..